### DISCUSSION TITLE
Orders not marked as tracked

### DISCUSSION THREAD

**Discussion by assaflei** [etsy/open-api#660] [Q&A] [answered]
We are updating tracking details of orders using the [createReceiptShipment](https://developer.etsy.com/documentation/reference/#operation/createReceiptShipment) api and the details are updating as expected

Some customers approached us with a question regarding behavior in their UI which I couldn't find any reference from the API
When going to an order tracking section, the order is marked as No tracking (this is the tracking status shown on the screen).
Than they press the Edit tracking button, save without any changes and the order is suddenly marked as Tracked.
It looks like it reconnects with the carrier tracking systems after the save action from the UI

Is there a way to make it work without the UI action?
Below found an image with the state of the order screen before entering the Edit tracking dialog

Any ideas how this can be resolved?

![tracking](https://user-images.githubusercontent.com/15898926/201918544-d42f38e3-a23d-42ea-95be-151a58245c6a.png)

**Comment by nateswart** [comment]
We looked into this basic issue awhile back and it appears to be connected to caching, specifically if the user already has the page open and is testing/watching (so has already generated a cached version of the page without the updated data). 

I'd be curious if the page _never_ updates without edit/saving, or if that is only necessary for the first 15-30 mins after the API call is made.

**Comment by assaflei** [comment]
Our updates occur without relation to the customer using the UI
It happens to several customers at different times and it doesn't seem to update until they will perform the manual save

Is there something that can be done from API point of view?
Is it only a UI issue or does it prevent the order from progressing?

**Comment by nateswart** [comment, marked as answer]
This appears to only be a UI issue (specifically, caching and from outside the API). 

If you find more examples of this, please share. Ideally,  they will be examples where the user has not clicked edit/save so that we can see the issue in real-time.

**Comment by assaflei** [comment]
Thanks @nateswart since my customer didn't return with additional details and examples I will mark the discussion as answered
In case I will have further details, I will follow up here
